source/geod24/LocalRest.d

```diff
@@ -502,9 +505,20 @@ public final class RemoteAPI (API, alias S = VibeJSONSerializer!()) : API
                     }
                 }
             });
-        catch (Exception e)
-            if (e !is exc)
-                throw e;
+        }
+        catch (Throwable t)
+        {
+            // We use this exception to exit the event loop
+            if (t is exc)
+                return;
+
+            import core.stdc.stdio, std.stdio;
+            printf("#### INTERNAL ERROR: %.*s\n", cast(int) t.msg.length, t.msg.ptr);
+            printf("Please file a bug at https://github.com/Geod24/localrest/\n");
+
+            try writeln("Full error: ", t);
+            catch (Exception e) { /* Nothing more we can do at this point */ }
+        }
```

Comment:
> Instead of silently loosing a thread, we catch `Throwable` and at least print a message before re-throwing

But the exception is not re-thrown here. Would it makes sense to add an `assert(0)` after the printouts?

Reply:
Well re-throwing doesn't do anything. The thread just disappear, without any output.
The only case where an `assert(0)` would make sense, IMO, would be:
- When compiled in `-release`
- When compiled with `-checkaction=halt`

Since localrest is a testing framework, the release part doesn't make much sense. For the second case, changing `checkaction`, well we won't get here to begin with, because we don't throw Exceptions, we only arrive here if there's an `AssertError` (or maybe other errors, e.g. `RangeError`).